hawkbit-rest-core/pom.xml

```diff
@@ -48,7 +48,6 @@
       <dependency>
          <groupId>javax.servlet</groupId>
          <artifactId>javax.servlet-api</artifactId>
-         <scope>provided</scope>
```

Comment:
why we removed the `provided` scope of the servlet-api? Can you explain the case?